.github/workflows/build-and-publish-debug.yml

```diff
@@ -25,4 +25,5 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
           buildtype: Debug
+          latest: 1
```

Comment:
We will need to ensure that this flag is set only in the branch that targets the current minor release. When a new minor release is published, we'll need to remove this from the old one (and set it in the new one)